sdk/build-tools/build.ts

```diff
@@ -0,0 +1,156 @@
+import { execSync } from "child_process";
```

Comment:
I think this whole directory can be removed since you added the `scripts` folder?

Reply:
Whoops not. I accidently left both files as I was testing where to run this from.

Do you think it's important for the "build" script to be part of the sdk subdir, or is it ok being run from the "scripts" folder on root?

Reply:
yeah, running from scripts is fine.  Making it it's own package made sense in the big sui mono-repo, but isn't necessary here